generate_stats_filename.py

```diff
@@ -0,0 +1,73 @@
+#This file is the script that users will actually run to generate the full set of statistics for a file
```

Comment:
I don't understand why this file is here?

Reply:
I was playing around with it before I implemented argparse, I don't know why it's still here.